_schemaV2.graphql

```diff
@@ -7757,6 +7757,9 @@ type Query {
 
     # Limit by published status.
     published: Boolean = true
+
+    # Returns sales the user has registered for if true, returns sales the user has not registered for if false.
+    registered: Boolean
```

Comment:
Similar to my comments in https://github.com/artsy/gravity/pull/13354 about the ambiguity of true/false with a query, we could address that here with a `filter: SaleFilter` and `enum SaleFilter { REGISTERED NOT_REGISTERED }`.

However I see this would probably break from precedent as with the `published` flag above, which was actually the original point of my confusion with it. maybe something like `filters: [SaleFilter]` is what I'm getting at, but clearly out of scope.

Reply:
I actually personally had the same confusion as what @erikdstock raised, and would like to use some brain power to make it more straightforward.